src/main/java/com/todoist/markup/EmojiParser.java

```diff
@@ -63,12 +63,15 @@ static synchronized void init() throws IOException {
             StringBuilder patternBuilder = new StringBuilder();
 
             // Pattern for standard emojis.
+            patternBuilder.append("(");
             patternBuilder.append(REGEXP_STANDARD_EMOJI);
 
             for(String shortcut : todoistShortcuts.keySet()) {
                 patternBuilder.append("|");
                 patternBuilder.append(Pattern.quote(shortcut));
             }
+            // Exclude links ://
```

Comment:
Can we improve this comment by stating exactly what happens if we don't do this?